src/CachePoolTest.php

```diff
@@ -738,7 +724,6 @@ public function testIsHitDeferred()
         $item = $this->cache->getItem('key');
         $item->set('value');
         $this->cache->saveDeferred($item);
-        $this->assertTrue($item->isHit());
```

Comment:
There was some discussion about this at https://groups.google.com/forum/?hl=en-GB#!topic/php-fig/pxy_VYgm2sU